etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/FilePartnerMetadataStorage.java

```diff
@@ -42,11 +43,18 @@ public static FilePartnerMetadataStorage getInstance() {
     }
 
     @Override
-    public PartnerMetadata readMetadata(final String uniqueId) throws PartnerMetadataException {
+    public Optional<PartnerMetadata> readMetadata(final String uniqueId)
+            throws PartnerMetadataException {
         Path filePath = getFilePath(uniqueId);
         try {
+            if (!Files.exists(filePath)) {
+                logger.logWarning("Metadata file not found: " + filePath);
```

Comment:
Let's update the `logWarning` to accept parameters so we can use `{}`.

Reply:
Done